grammar.js

```diff
@@ -701,7 +702,11 @@ const rules = {
       choice(
         // Make a single-parameter lambda node look like any other lambda node.
         alias($._single_parameter_parameters, $.parameters),
-        seq($.parameters, opt(seq(':', field('return_type', $._type)))),
+        seq(
+          $.parameters,
+          opt($.capability_list),
+          opt(seq(':', field('return_type', $._type)))
```

Comment:
Nitpick: Based on this config settting, there should be a trailing comma here. https://github.com/slackhq/tree-sitter-hack/blob/d434b4e6c9af0524b163d7a912c8ce79d2b1f778/.prettierrc#L6 

Not blocking approval, but might check on overrides in editor config settings or make sure prettier's enabled to prevent unnecessary noise in the future with bigger diffs.